package.json

```diff
@@ -31,16 +31,10 @@
     "name": "The MongoDB NodeJS Team",
     "email": "dbx-node@mongodb.com"
   },
-  "peerOptionalDependencies": {
-    "kerberos": "^1.1.0",
-    "mongodb-client-encryption": "^1.0.0",
-    "snappy": "^6.1.1",
-    "bson-ext": "^2.0.0"
-  },
```

Comment:
This is unused IIUC

Reply:
I'm a bit confused, isn't this what specifies the compatible versions of these dependencies? e.g., for us it's snappy 6 or 7?

Reply:
only implicitly in the sense that it happens to have some versions numbers which are kinda accurate (bson-ext is wrong). The `peerOptionalDependencies` was used in 3.5 by our old [require_optional dependency](https://github.com/christkv/require_optional/blob/c11c4ec54c29d9f5b8a335db379a4670ceeb6cfb/index.js#L51). It's not a standard field and it isn't respected by any npm install logic, so I think dropping it, maybe in favor of a support table in our readme or docs would be better, thoughts?

Reply:
do we have an optional dependencies file that's a single point of entry for all these? because that's where I'd want to specify these; alternatively, I suppose a table in the README for an optional dependencies section would not be out of place since I guess this is how we tell users these optional customizations? (let's not include bson-ext there at all though, it has worse performance than js-bson)

Reply:
I made NODE-3562 which recommends something more than a table in our README, but could be used to track that, sound good?

Reply:
yeah, looks good, tyvm!